internal/exec/util/passwd.go

```diff
@@ -244,8 +255,19 @@ func (u Util) SetPasswordHash(c types.PasswdUser) error {
 	return err
 }
 
-// CreateGroup creates the group as described.
-func (u Util) CreateGroup(g types.PasswdGroup) error {
+// CreateGroup creates the group as described. If shouldExist variable
+// is set to false and the group already exists then it will be deleted.
+func (u Util) CreateGroup(g types.PasswdGroup, shouldExist bool) error {
+	if !shouldExist {
```

Comment:
Does `groupdel` not error out if it's called to delete a group that doesn't exist?

Reply:
@arithx ahh... I see your point. How about adding another function to check if the group exists or not?